privateCA/create_certs.sh

```diff
@@ -0,0 +1,273 @@
+    -days "$CERT_VALIDITY_DAYS" -out "$CERTS_DIR/$key_name.pem" -in "$CERTS_DIR/$key_name.csr" \
+    -notext # don't output the text form of a certificate to the output file
+}
+
+
+function generate_ca_signed_cert {
+    local readonly key_name="$1"
+    local readonly cert_subject="$2"
+
+    create_csr "$key_name" "$cert_subject"
+    sign_csr_with_ca "$key_name"
+}
+
+
+function generate_client_cert {
+    local readonly name="$1"
+
+    local readonly description="-apigee-client-cert"
+    generate_key "$name"
+    generate_ca_signed_cert "$name" "$description"
+    convert_cert_to_der "$name"
+}
+
+echo "Going to create mutual TLS certs with these details"
+echo "AWS_PROFILE: ${AWS_PROFILE}"
+echo "CERT_PREFIX: ${CERT_PREFIX}"
+echo "SECRET_OUTPUT_PREFIX: ${SECRET_OUTPUT_PREFIX}"
+echo "BUCKET_PREFIX: ${BUCKET_PREFIX}"
+echo "DRY_RUN: ${DRY_RUN}"
+read -p "Press any key to resume or press ctrl+c to exit ..."
+
+# Recreate output dirs
+rm -rf "$CERTS_DIR" "$KEYS_DIR" "$CRL_DIR" "$CONFIG_DIR"
+mkdir "$CERTS_DIR" "$KEYS_DIR" "$CRL_DIR" "$CONFIG_DIR" "$BACKUP_CERTS_DIR" "$BACKUP_KEYS_DIR"
+
+
+# Create database and serial files
+touch "$CONFIG_DIR/index.txt"
+echo '1000' > "$CONFIG_DIR/crlnumber.txt"
+echo '01' > "$CONFIG_DIR/serial.txt"
+
+# Generate CA key and self-signed cert
+echo "Generating CA credentials..."
+generate_key "$CA_NAME"
+generate_ca_cert "$CA_NAME"
+
+generate_client_cert "apigee_client_cert"
+generate_client_cert "apigee_client_cert_sandbox"
+
+CA_KEY_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'CAKeySecret`].OutputValue' --output text)
+CA_CERT_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'CACertSecret`].OutputValue' --output text)
+CLIENT_KEY_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'ClientKeySecret`].OutputValue' --output text)
+CLIENT_CERT_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'ClientCertSecret`].OutputValue' --output text)
+CLIENT_SANDBOX_KEY_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'ClientSandboxKeySecret`].OutputValue' --output text)
+CLIENT_SANDBOX_CERT_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'ClientSandboxCertSecret`].OutputValue' --output text)
+TRUSTSTORE_BUCKET_ARN=$(aws cloudformation describe-stacks \
+    --stack-name account-resources \
+    --query 'Stacks[0].Outputs[?OutputKey==`'"${SECRET_OUTPUT_PREFIX}"'TrustStoreBucket`].OutputValue' --output text)
```

Comment:
you dont need SECRET_OUTPUT_PREFIX here - we have one bucket that all CA certificates will go to

Reply:
The SECRET_OUTPUT_PREFIX has been removed.